simbad/command_line/__init__.py

```diff
@@ -385,8 +385,10 @@ def _simbad_contaminant_search(args):
         logger.debug("Contaminant MR summary file: %s", mr_summary_f)
         molecular_replacement.summarize(mr_summary_f)
 
+        make_output_dir(stem, os.path.join(args.work_dir, 'output_files'), mr_summary_f, args.mr_program)
```

Comment:
Do this `os.path.join` operation only once and save it to a variable ... it makes changing the `dir` name easier later on

Reply:
Same obviously applies to the other functions where you use it.